src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocParagraphCheck.java

```diff
@@ -181,7 +181,10 @@ private void checkParagraphTag(DetailNode tag) {
         else if (newLine == null || tag.getLineNumber() - newLine.getLineNumber() != 1) {
             log(tag.getLineNumber(), MSG_LINE_BEFORE);
         }
-        if (allowNewlineParagraph && isImmediatelyFollowedByText(tag)) {
+        if (!allowNewlineParagraph && isImmediatelyFollowedByNewLine(tag)) {
+            log(tag.getLineNumber(), MSG_MISPLACED_TAG);
+        }
+        if (isImmediatelyFollowedByText(tag)) {
```

Comment:
This second `if` condition is a separate condition, not connected to the previous via `else`. Is it expected? If yes, it looks odd to me because, in this case, the same message is logged twice when both `!allowNewlineParagraph` and `isImmediatelyFollowedByText(tag)` are true.

Reply:
hard to say .... this Check is very very broken, two more issues (see above) are required to be fixed to let Check be clear   in what it is doing.
As this this code does not make any effect, lets proceed, we will have final implementation at https://github.com/checkstyle/checkstyle/issues/15685

Reply:
I'm good to proceed, considering those conditions are different, and we shouldn't have a situation when both are true.